### PR TITLE
Test bridge burn

### PR DESCRIPTION
**Pull request by iulianpascalau**
- Changed integration test for the ESDT burn edge case, added new asserts
- Fixed a typo

### CODE REVIEW

**Review by sasurobert — DISMISSED** [review]

File: integrationTests/multiShard/smartContract/polynetworkbridge/bridge_test.go [hunk 1]
```diff
@@ -102,18 +105,65 @@ func TestBridge_Setup(t *testing.T) {
...
 	require.NotNil(t, vmOutput)
-	fmt.Println()
-	fmt.Println(vmOutput.ReturnData)
 	require.NotZero(t, len(vmOutput.ReturnData[0]))
-	require.Equal(t, []byte("WEGLD"), vmOutput.ReturnData[0][:4])
+
+	tokenIdentifier := vmOutput.ReturnData[0]
+	require.Equal(t, []byte("WEGLD"), tokenIdentifier[:5])
+
+	burntValue := big.NewInt(5)
```
Comment (bogdan-rosianu) [inline integrationTests/multiShard/smartContract/polynetworkbridge/bridge_test.go:124]: `valueToBurn` ? It is not burnt yet when declaring the variable

File: integrationTests/vm/esdt/esdtProcess_test.go [hunk 2]
```diff
@@ -80,22 +80,22 @@ func TestESDTIssueAndTransactionsOnMultiShardEnvironment(t *testing.T) {
 	nonce, round = integrationTests.WaitOperationToBeDone(t, nodes, nrRoundsToPropagateMultiShard, nonce, round, idxProposers)
 	time.Sleep(time.Second)
 
-	tokenIdenfitifer := string(getTokenIdentifier(nodes))
+	tokenIdentifier := string(getTokenIdentifier(nodes))
```
Comment (bogdan-rosianu) [inline integrationTests/vm/esdt/esdtProcess_test.go:83]: also, please rename `initalSupply` to `initialSupply` inside `esdtProcess_test.go` file

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: integrationTests/vm/esdt/esdtProcess_test.go (hunk 2 above)
Comment (iulianpascalau) [reply integrationTests/vm/esdt/esdtProcess_test.go:83]: So many typos :)

File: integrationTests/multiShard/smartContract/polynetworkbridge/bridge_test.go (hunk 1 above)
Comment (iulianpascalau) [reply integrationTests/multiShard/smartContract/polynetworkbridge/bridge_test.go:124]: changed

File: integrationTests/vm/esdt/esdtProcess_test.go (hunk 2 above)
Comment (iulianpascalau) [reply integrationTests/vm/esdt/esdtProcess_test.go:83]: fixed

**Review by bogdan-rosianu — APPROVED** [review]

**Review by mateigraura — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]
